packages/dredd-transactions/package.json

```diff
@@ -29,6 +30,7 @@
     "fury-adapter-apib-parser": "0.16.0",
     "fury-adapter-oas3-parser": "0.9.1",
     "fury-adapter-swagger": "0.27.2",
+    "rimraf": "3.0.0",
```

Comment:
This should still be a dev dep

Reply:
Hm, per eslint `rimraf` should be in dependencies:

```
/home/circleci/project/packages/dredd-transactions/scripts/prepack.js
  7:16  error  'rimraf' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
```

Reply:
Which is clearly a mistake, since ESLint doesn't care what for the package is required. I suggest we disable eslint rule for `rimraf` in this case.